lib/node_modules/@stdlib/math/base/special/asec/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@stdlib/math/base/special/asec",
+  "version": "0.0.0",
+  "description": "Compute the inverse (arc) secant of a number.",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "The Stdlib Authors",
+    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+  },
+  "contributors": [
+    {
+      "name": "The Stdlib Authors",
+      "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+    }
+  ],
+  "main": "./lib",
+  "directories": {
+    "benchmark": "./benchmark",
+    "doc": "./docs",
+    "example": "./examples",
+    "lib": "./lib",
+    "scripts": "./scripts",
```

Comment:
This package does not appear to have a `scripts` folder. This entry can be removed.